RPCTests/scripts/tests/MineBlocks2/step00_node02_mineBlocks.js

```diff
@@ -0,0 +1,31 @@
+process.stdout.write("TEST_mineBlockOnNode2 ");
+
+var latestBlock;
+web3.eth.getBlockNumber(function(err, res){ onGetBlockNumber1(err, res); })
+onGetBlockNumber1 = function (err, res)
+{
+    latestBlock = res;
+    web3.test.mineBlocks(1, function(err, res){ onResult(err, res); })
+}
+
+
+onResult = function (err,res) 
+{
+   function sleep(ms) {
+      return new Promise(resolve => setTimeout(resolve, ms));
+   }
+
+   //wait for block being mined and propagated
+   sleep(1000).then(() => {
+	web3.eth.getBlockNumber(function(err, res){ onGetBlockNumber(err, res); })
+   });
+}
+
+onGetBlockNumber = function (err, res)
+{
+   if (res == latestBlock + 1)
```

Comment:
instead of check for failure like this, we could use `tape`
in `package.json` add  "tape": "^4.6.0",
in this file: 
`var tape = require('tape')`

and

```
tape('test suite', function (t) {
  t.test('test name', function (st) {
    st.equal(res, latestBlock + 1)
  })
}